build/validate-bicep.sh

```diff
@@ -30,7 +30,7 @@ do
         exec 3>&-
     fi
     
-    if [[ ! $EXITCODE -eq 0 || (! -z $STDERR && ! $STDERR == $WARNING_MSG* ) ]]
+    if [[ ! $EXITCODE -eq 0 || (! -z $STDERR && ! $STDERR == $WARNING_MSG* && ! $STDERR == *"Error"* ) ]]
```

Comment:
This was a small bug. It'd catch all the warning messages but would miss any errors that occurred after that. So this adds a check for errors